contracts/assetListing/AssetListingGUni.sol

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.7.5;
+pragma abicoder v2;
+
+import {IAaveOracle} from "./interfaces/IAaveOracle.sol";
+
+interface IPoolConfigurator {
+  struct InitReserveInput {
+    address aTokenImpl;
+    address stableDebtTokenImpl;
+    address variableDebtTokenImpl;
+    uint8 underlyingAssetDecimals;
+    address interestRateStrategyAddress;
+    address underlyingAsset;
+    address treasury;
+    address incentivesController;
+    string underlyingAssetName;
+    string aTokenName;
+    string aTokenSymbol;
+    string variableDebtTokenName;
+    string variableDebtTokenSymbol;
+    string stableDebtTokenName;
+    string stableDebtTokenSymbol;
+    bytes params;
+  }
+
+  function batchInitReserve(InitReserveInput[] calldata input) external;
+
+  function configureReserveAsCollateral(
+    address asset,
+    uint256 ltv,
+    uint256 liquidationThreshold,
+    uint256 liquidationBonus
+  ) external;
+
+  function setReserveFactor(address asset, uint256 reserveFactor) external;
+}
+
+interface IProposalIncentivesExecutor {
+  function execute() external;
+}
+
+contract AssetListingGUni is IProposalIncentivesExecutor {
+  address constant POOL_CONFIGURATOR = 0x23A875eDe3F1030138701683e42E9b16A7F87768;
+  address constant AAVE_ORACLE = 0xA50ba011c48153De246E5192C8f9258A2ba79Ca9;
+
+  // BPT V2 to list
```

Comment:
Could you rewrite the comments and replace `BPT` with `GUNI` please? Same with `Balancer` and so

Reply:
👍